lib/fxpay/index.js

```diff
@@ -56,13 +56,20 @@
 
 
   exports.purchase = function _purchase(productId, onPurchase, opt) {
-    opt = opt || {};
-    opt.maxTries = opt.maxTries || undefined;
-    opt.pollIntervalMs = opt.pollIntervalMs || undefined;
+    opt = utils.defaults(opt, {
```

Comment:
I think that's fine. I was listing these properties as undefined merely to document them because it seemed helpful to know what properties are *valid* on an options argument when looking at the top of a function definition. In other words: sigh, javascript.